torch/onnx/symbolic_opset9.py

```diff
@@ -2248,33 +2248,58 @@ def topk(g, self, k, dim, largest, sorted, out=None):
 
 
 def to(g, self, *args):
-    # ONNX doesn't have a concept of a device, so we ignore device casts
-    if len(args) == 4:
-        if args[0].node().kind() == "prim::device" or args[0].type().isSubtypeOf(ListType.ofInts()):
+
+    def is_aten_to_device_only(*args):
+        if len(args) == 4:
             # aten::to(Tensor, Device, bool, bool, memory_format)
-            return self
+            if args[0].node().kind() == "prim::device" or \
```

Comment:
Instead of:
```python
if x:
  return True
if y:
  return True
```
write:
```python
return x or y
```

It's more compact and just as clear

Reply:
This is not what I meant.

I meant to remove the if statement entirely and just have a single return statement. Something like:

```python
return (args[0].node().kind() == "prim::device" or
            args[0].type().isSubtypeOf(ListType.ofInts() or
            (sym_help._is_value(args[0]) and
             args[0].node().kind() == "onnx::Constant" and
             isinstance(args[0].node()["value"], str))
```